code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -1117,6 +1118,7 @@
 	allow_no_queen_actions = TRUE
 	allow_queen_evolve = FALSE
 	ignore_slots = TRUE
+	no_latejoin_burrowed = TRUE
```

Comment:
This is default FALSE, and only for yatuja do you set this true. So I don't think your description "Corrupted and other hives can no more grow without caps in containment" is currently accurate. Did you intend to also set this true for other hives?

Reply:
Well. I thought about it. But i only know it surely need for Hellhound pack, and probably corrupted hive. Since they are usually appear in round. Other's hive... don't have it. But you right. Probably i need to turn it for every hive except normal.